extension/src/config/init/ensure-payment-custom-type.js

```diff
@@ -1,4 +1,4 @@
-const paymentCustomType = require('../../../resources/payment-custom-types.json')
+const paymentCustomType = require('../../../resources/web-components-payment-type')
```

Comment:
It is not a JSON anymore?

Reply:
Yes, it is JSON.

Reply:
Then why not web-components-payment-type.json?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/233/commits/7f3c602c8fa78bea41e270287bfe710dcbad954c